code/controllers/subsystem/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -994,6 +994,7 @@
 	player_mind.set_assigned_role(SSjob.get_job_type(/datum/job/voidwalker))
 	player_mind.special_role = antag_flag
 	player_mind.add_antag_datum(antag_datum)
+	ADD_TRAIT(voidwalker, TRAIT_STEALTHY_INTERACT, INNATE_TRAIT)
```

Comment:
Should they really be hiding all cog wheels forever? Shouldn't they only be hiding them when they are stealthed? Like "in space"?

Reply:
yeah that's a good idea